src/java.base/share/classes/jdk/internal/misc/CDS.java

```diff
@@ -51,9 +51,9 @@
     public static native long getRandomSeedForDumping();
 
     /**
-     * Check if dynamic dumping is enabled via the DynamicDumpSharedSpaces flag.
+     * Check if CDS dumping is enabled via the DynamicDumpSharedSpaces or the DumpSharedSpaces flag.
      */
-    public static native boolean isDynamicDumpingEnabled(); // will return false for static dumping.
+    public static native boolean isCDSDumpingEnabled();
```

Comment:
I suggest to rename `CDS::isCDSDumpingEnabled` to `CDS:isDumpingEnabled` as this method is a static method in `CDS` case and the word `CDS` in the method name is just noise.

JVM entry point `JVM_IsCDSDumpingEnabled`  is good.

Reply:
Fixed in webrev 03.